format/binary.md

```diff
@@ -446,6 +446,21 @@ The **correct** interpretation of this data, with accumulation, is:
 **Type ID 0x18**
 
 ### PhysicalProperties
+
+The `PhysicalProperties` type is a struct composed of five f32s:
+
+| Field Name       | Format | Value                                                        |
+|:-----------------|:-------|:-------------------------------------------------------------|
+| Density          | f32    | The density set for the custom physical properties           |
+| Friction         | f32    | The friction set for the custom physical properties          |
+| Elasticity       | f32    | The elasticity set for the custom physical properties        |
+| FrictionWeight   | f32    | The friction weight set for the custom physical properties   |
+| ElasticityWeight | f32    | The elasticity weight set for the custom physical properties |
```

Comment:
Maybe more specifically, this is the `CustomPhysicalProperties` value, and `PhysicalProperties` is a type that contains a flag that may be followed by a `CustomPhysicalProperties` value.